com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/IOSBundler.java

```diff
@@ -499,7 +527,14 @@ public void bundleApplication(Project project, File bundleDir, ICanceled cancele
         BundleHelper.throwIfCanceled(canceled);
         File zipFile = new File(bundleDir, title + ".ipa");
         File zipFileTmp = new File(bundleDir, title + ".ipa.tmp");
-        processBuilder = new ProcessBuilder("zip", "-qr", zipFileTmp.getAbsolutePath(), payloadDir.getName());
+
+        // NOTE: We replaced the java zip file implementation(s) due to the fact that XCode didn't want
+        // to import the resulting zip files.
+        if (swiftSupportDir.exists())
+            processBuilder = new ProcessBuilder("zip", "-qr", zipFileTmp.getAbsolutePath(), payloadDir.getName(), swiftSupportDir.getName());
```

Comment:
Is zip guaranteed to exist?

Reply:
It might not on all platforms. But on macOS we've used this since 2016.
We may change back to Java zip now if version 11 is compatible with the Apple toolchain (it wasn't back then)